packages/wonder-blocks-modal/src/components/__docs__/modal-panel.stories.js

```diff
@@ -0,0 +1,459 @@
+            {`Lorem ipsum dolor sit amet, consectetur
+adipiscing elit, sed do eiusmod tempor incididunt
+ut labore et dolore magna aliqua. Ut enim ad minim
+veniam, quis nostrud exercitation ullamco laboris
+nisi ut aliquip ex ea commodo consequat. Duis aute
+irure dolor in reprehenderit in voluptate velit
+esse cillum dolore eu fugiat nulla pariatur.
+Excepteur sint occaecat cupidatat non proident,
+sunt in culpa qui officia deserunt mollit anim id
+est.`}
+        </Body>
+        <br />
+        <Body>
+            {`Lorem ipsum dolor sit amet, consectetur
+adipiscing elit, sed do eiusmod tempor incididunt
+ut labore et dolore magna aliqua. Ut enim ad minim
+veniam, quis nostrud exercitation ullamco laboris
+nisi ut aliquip ex ea commodo consequat. Duis aute
+irure dolor in reprehenderit in voluptate velit
+esse cillum dolore eu fugiat nulla pariatur.
+Excepteur sint occaecat cupidatat non proident,
+sunt in culpa qui officia deserunt mollit anim id
+est.`}
+        </Body>
+        <br />
+        <Body>
+            {`Lorem ipsum dolor sit amet, consectetur
+adipiscing elit, sed do eiusmod tempor incididunt
+ut labore et dolore magna aliqua. Ut enim ad minim
+veniam, quis nostrud exercitation ullamco laboris
+nisi ut aliquip ex ea commodo consequat. Duis aute
+irure dolor in reprehenderit in voluptate velit
+esse cillum dolore eu fugiat nulla pariatur.
+Excepteur sint occaecat cupidatat non proident,
+sunt in culpa qui officia deserunt mollit anim id
+est.`}
+        </Body>
+    </>
+);
+
+export default {
+    title: "Modal/Building Blocks/ModalPanel",
+    component: ModalPanel,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        docs: {
+            description: {
+                component: null,
+            },
+            source: {
+                // See https://github.com/storybookjs/storybook/issues/12596
+                excludeDecorators: true,
+            },
+        },
+        decorators: [
+            (Story: any): React.Element<typeof View> => (
+                <View style={styles.example}>{Story()}</View>
+            ),
+        ],
+        viewport: {
+            viewports: customViewports,
+            defaultViewport: "desktop",
+        },
+        chromatic: {
+            viewports: [320, 640, 1024],
+        },
+    },
+    // Make the following props null in the control panel
+    // because setting an object for a React node is
+    // not practical to do manually, and adding a React Element
+    // as one of the default args just results in an incoherent
+    // json object in the control panel.
+    argTypes: {
+        content: {
+            control: {type: null},
+        },
+        header: {
+            control: {type: null},
+        },
+        footer: {
+            control: {type: null},
+        },
+    },
+};
+
+export const Default: StoryComponentType = (args) => (
+    <View style={styles.previewSizer}>
```

Comment:
**nit (non-blocking):** Not sure if you know this already..... as I see that these two views are being reused in all the stories (`previewSizer` and `modalPositioner`), we could also use the `decorator` setting in the default export so we could create that common wrapper for all stories in this file. No need to do it here, but might come handy in future PRs.

e.g.

```
export default {
  ...
  decorators: [
        (Story: StoryComponentType): React.Element<typeof View> => (
            <View style={styles.previewSizer}>
                <View style={styles.modalPositioner}>
                    {Story()}
               </View> 
            </View>
        ),
    ],
};

This also helps to avoid including these two decorative wrappers into the "Show code" tab.
```